src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -1467,3 +1313,222 @@ def _InitLib(self):
 
             self._dmLib.pychip_DeviceController_GetLogFilter = [None]
             self._dmLib.pychip_DeviceController_GetLogFilter = c_uint8
+
+            self._dmLib.pychip_OpCreds_AllocateController.argtypes = [c_void_p, POINTER(
+                c_void_p), c_uint64, c_uint64, c_uint16, c_char_p, c_bool, c_bool, POINTER(c_uint32), c_uint32, c_void_p]
+            self._dmLib.pychip_OpCreds_AllocateController.restype = PyChipError
+
+            self._dmLib.pychip_OpCreds_AllocateControllerForPythonCommissioningFLow.argtypes = [
+                POINTER(c_void_p), c_void_p, c_char_p, c_uint32, c_char_p, c_uint32, c_char_p, c_uint32, c_char_p, c_uint32, c_uint16, c_bool]
```

Comment:
c_char_p in https://docs.python.org/3/library/ctypes.html is documented as `(NUL terminated)`. should we just say c_void_p? Not sure if it matters though.

Reply:
Updated to POINTER(c_char), (but `POINTER(c_uint8_t)` won't work)